--- elasticsearch-py-1.5.0/setup.py~	2015-05-18 10:03:00.000000000 -0400
+++ elasticsearch-py-1.5.0/setup.py	2015-06-23 17:43:33.729842778 -0400
@@ -31,8 +31,8 @@
 if sys.version_info[0] == 2:
     # only require thrift if we are going to use it
     if os.environ.get('TEST_ES_CONNECTION', None) == 'ThriftConnection':
-        tests_require.append('thrift==0.9.1')
-    tests_require.append('pylibmc==1.4.1')
+        tests_require.append('thrift')
+    tests_require.append('pylibmc')
 
 setup(
     name = 'elasticsearch',
